api/organization.go

```diff
@@ -29,8 +30,9 @@ func (r ListOrganizationsRequest) ValidationRules() []validate.ValidationRule {
 }
 
 type CreateOrganizationRequest struct {
-	Name   string `json:"name"`
-	Domain string `json:"domain"`
+	Name                string   `json:"name"`
+	Domain              string   `json:"domain"`
+	AllowedLoginDomains []string `json:"allowedLoginDomains" note:"domains which can be used with social login for this organization" example:"['example.com', 'infrahq.com']"`
```

Comment:
Does this need to be here for creation?

Reply:
I thought it may be useful when we directly create an org so that we dont have to configure a username/password for that org. I removed it for now though, we can add it when we need it.